Source/JavaScriptCore/runtime/NumericStrings.h

```diff
@@ -74,48 +97,43 @@ class NumericStrings {
 
     void clearOnGarbageCollection()
     {
-        for (auto& entry : intCache)
-            entry.jsString = nullptr;
-        for (auto& entry : smallIntCache)
+        for (auto& entry : m_intCache)
             entry.jsString = nullptr;
+        // 0-9 are managed by SmallStrings. They never die.
+        for (unsigned i = 10; i < m_smallIntCache.size(); ++i)
+            m_smallIntCache[i].jsString = nullptr;
     }
 
-private:
-    static const size_t cacheSize = 64;
-
-    template<typename T>
-    struct CacheEntry {
-        T key;
-        String value;
-    };
+    template<typename Visitor>
+    void visitAggregateImpl(Visitor& visitor)
```

Comment:
nit: I think you can just call this `visitAggregate` because unlike other objects, you're not going to use this to instantiate `visitAggregate` methods.

Reply:
Changed.